src/sentry/utils/snuba.py

```diff
@@ -188,6 +188,9 @@ def log_snuba_info(content):
     # message also maps to span description but gets special handling
     # to support wild card searching by default
     "message": "name",
+    # These sample columns are for debugging only and shouldn't be used
+    "sampling_weight": "sampling_weight",
+    "sampling_factor": "sampling_factor",
```

Comment:
These act as hidden columns and only work if queried against like the rest of our hidden columns, yeah?

Reply:
yep